deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -15,8 +15,16 @@
             "host": {
                 "sourcePath": "/media/nvme0n1/pgdata"
             }
+        }, {
+            "name": "tmp",
+            "host": {
+                "sourcePath": "/media/nvme0n1/tmp"
+            }
         }],
         "environment": [{
+            "name": "NB_MAX_TRIP_DISTANCE",
+            "value": "2680"
```

Comment:
After looking at this again, I disagree. Both NB_MAX_TRIP_DISTANCE and NB_BOUNDARY_BUFFER are used in multiple files, and the value of those variables should be consistent in all files. 

What if instead I moved the declarations of these two variables to run_analysis.sh and `export`ed them so they're available everywhere else, removing their definitions from each of the individual scripts they're used in?